astropy/io/fits/tests/test_table.py

```diff
@@ -1239,7 +1239,7 @@ def test_assign_multiple_rows_to_table(self):
         assert tbhdu2.columns.columns[2].array[0] == ''
         assert (tbhdu2.columns.columns[3].array[0] ==
                 np.array([0., 0., 0., 0., 0.], dtype=np.float32)).all()
-        assert tbhdu2.columns.columns[4].array[0] == True  # noqa
+        assert tbhdu2.columns.columns[4].array[0] is np.bool_(True)
```

Comment:
@mhvk do you think this is better, or would something akin to your suggestion here https://github.com/astropy/astropy/pull/13754#discussion_r983622791 be better?

Reply:
This can just be `np.True_`.

But, yes, I like my suggestion perhaps a little better as it makes clear that the intent of the test is to verify that type of the output from `__eq__` -- and if `numpy` ever where to return python bool scalars, that would be fine!